otterdog/eclipse-lmos.jsonnet

```diff
@@ -1,5 +1,21 @@
 local orgs = import 'vendor/otterdog-defaults/otterdog-defaults.libsonnet';
 
+local customRuleset(name) = 
```

Comment:
I would reflect in the name that it targets the default branch, otherwise someone might use that template for other branches as well but it wont work as expected, but that is a minor nit.

Reply:
Better now?